contentcuration/contentcuration/frontend/settings/views/Account/index.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+
+  <div>
+    <h2 class="heading">Basic Information</h2>
+    <div>
+
+    </div>
+    <KFixedGrid numCols="8" gutter="40">
+      <KFixedGridItem span="2" class="row">
+        <b>Username</b>
+      </KFixedGridItem>
+      <KFixedGridItem span="6" class="row">
+        {{ $store.state.session.currentUser.email }}
+      </KFixedGridItem>
+      <KFixedGridItem span="2" class="row">
+        <b>Full name</b>
+      </KFixedGridItem>
+      <KFixedGridItem span="6" class="row">
+        <span>
+          {{ fullName }}
+          <KButton appearance="basic-link" text="Edit" @click="showFullNameForm = !showFullNameForm" />
+        </span>
+      </KFixedGridItem>
+      <KFixedGridItem span="2" class="row">
+        <b>Password</b>
+      </KFixedGridItem>
+      <KFixedGridItem span="6" class="row">
+        <KButton appearance="basic-link" text="Change password" @click="showPasswordForm = !showPasswordForm" />
+      </KFixedGridItem>
+    </KFixedGrid>
+
+    <h2 class="heading">API Token</h2>
+    <p style='color: red'>You can use this code to...Ut aliquam ornare turpis, aliquam ornare purus gravida sed. Nullam imperdiet iaculis tincidunt. NEED COPY</p>
+    <CopyToken class="copy-token" :token="$store.state.session.currentUser.api_token" :hyphenate="false" />
+
+    <h2 class="heading-export">Export account data</h2>
+    <p>You will be sent an email with all information linked to your account</p>
+    <KButton text="Export data" primary @click="startDataExport" />
+
+    <h2 class="heading">Delete account</h2>
+    <div v-if="channelsAsSoleEditor.length > 0">
+      <p>You must delete these channels manually or invite others to edit them before you can delete your account.</p>
+      <p v-for="channel in channelsAsSoleEditor" :key="channel.id">
+        <!--
+          TODO: RESOLVE AND REMOVE THIS COMMENT
```

Comment:
@jayoshih Should this be sending users somewhere other than channels?

For context - this is where we list channels that the user is the sole owner of - meaning they cannot delete their account until they delete their channels or invite another person to also be an editor for it.

So - each channel is listed as a link. Is there currently or will there be a place where a channel can be deleted other than the ChannelIndex?

Reply:
It should link to the channel edit page. I've opened [an issue](https://github.com/learningequality/studio/issues/1889) though to add the ability to delete the channel from there (no need to add this in your PR though)